client/components/quizeditor/QuizHistoryDisplay.jsx

```diff
@@ -1,20 +1,77 @@
 import React from 'react';
 import { Button } from '@material-ui/core';
+import Table from '@material-ui/core/Table';
+import TableBody from '@material-ui/core/TableBody';
+import TableCell from '@material-ui/core/TableCell';
+import TableContainer from '@material-ui/core/TableContainer';
+import TableHead from '@material-ui/core/TableHead';
+import TableRow from '@material-ui/core/TableRow';
```

Comment:
Sometimes easier to put all of these in one e.g.

import {Table, TableBody, etc} from '@material-ui/core';